docs/nodes/blobstream-binary.mdx

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_label: Install the Blobstream binary
+description: This guide shows you how to install the Blobstream binary.
+---
+
+# Install the Blobstream binary
+
+import constants from "../constants/constants.js";
+import InlineText from "@site/src/components/InlineText.js";
+
+The [orchestrator](../blobstream-orchestrator) is the software that signs the
+Blobstream attestations, and the [relayer](../blobstream-relayer) is the one that
+relays them to the target EVM chain.
+
+## Install
+
+```mdx-code-block
+1. [Install Go](https://go.dev/doc/install) <InlineText constant={constants.golangBlobstream} />
+2. Clone the `https://github.com/celestiaorg/orchestrator-relayer` repository:
+
+    <pre><code>
+    git clone https://github.com/celestiaorg/orchestrator-relayer.git<br />
+    cd orchestrator-relayer<br />
+    git checkout {constants.orchrelayVersion}<br />
+    </code></pre>
+```
+
+3. Install the Blobstream CLI
+
+   ```sh
+   make install
+   ```
```

Comment:
```suggestion
```sh
make install
```
```

Reply:
I'm going to resolve this without committing suggestion, as the indentation was done by Prettier and[ formats the numbered list properly in the preview](https://docs.celestia.org/pr-preview/pr-1115/nodes/blobstream-binary/)

Reply:
cool :+1: